### ISSUE TITLE
[v2.0] keyboard shortcuts are not working in popup menu under Linux

### ISSUE THREAD

**Issue by anb0s** [anb0s/EasyShell#106]
The shortcut **Alt + E** opens a popup menu, but the described keys '0'-'9' etc. are not working, one has to execute mouse click! Also Enter is not working! Under Windows it works like expected.

**Comment by anb0s** [comment]
The shortcuts are working, but during Command-Execution we have a call to `HandlerUtil.getActivePart(event)` and it returns `null` at Linux:

```
public class Execute extends AbstractHandler {

    public Object execute(ExecutionEvent event) throws ExecutionException {
        IWorkbenchPart activePart = HandlerUtil.getActivePart(event);
        if (activePart != null) {
          ...
        } else {
        	Activator.logError("getActivePart() returns null", null);
        }
        return null;
    }

```

It works at Windows. It looks like an Eclipse platform bug described here:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=242246

**Comment by anb0s** [comment]
done

**Comment by anb0s** [comment]
included in build #111
